ember-cli-build.js

```diff
@@ -42,6 +43,8 @@ module.exports = function() {
       });
     }
     jsTree = stripGlimmerUtilities(jsTree);
+  } else {
+    jsTree = addGlimmerEnv(jsTree);
```

Comment:
I don't think we particularly want to babel these files twice (which is what it seems like is being done here), can we add the plugin to the existing babel instead?